src/languageSupport/monaco.utils.ts

```diff
@@ -0,0 +1,17 @@
+export const pointToCorrectWorkers = () => {
+  self.MonacoEnvironment.getWorkerUrl = (_moduleId: string, label: string) => {
+    if (label === 'json') {
+      return './json.worker.bundle.js'
+    }
+    if (label === 'css' || label === 'scss' || label === 'less') {
+      return './css.worker.bundle.js'
+    }
+    if (label === 'html' || label === 'handlebars' || label === 'razor') {
+      return './html.worker.bundle.js'
+    }
+    if (label === 'typescript' || label === 'javascript') {
+      return './ts.worker.bundle.js'
+    }
+    return './editor.worker.bundle.js'
```

Comment:
this needs a cache busting mechanism attached to it

Reply:
These files are from monaco-editor, pegged to a minor version (only patches allowed) -- because compatibility with the associated libs are dependent on minor versions.

Does this need to be cache busted?